src/subdomains/staking/application/services/liquidity-management.service.ts

```diff
@@ -260,7 +260,7 @@ export class LiquidityManagementService {
             });
           },
           updateFunc: (masternode: Masternode, txId: string) => {
-            console.info(`Sending collateral to masternode\n\towner: ${masternode.owner}\n\twith tx: ${txId}`);
+            console.info(`Sending collateral to masternode \n\towner: ${masternode.owner} \n\twith tx: ${txId}`);
```

Comment:
doesn't really matter, but why adding whitespace before a new-line?

Reply:
![image](https://user-images.githubusercontent.com/85513542/198223048-e47aad84-2180-4953-a826-cc3fbf21d361.png)
Application Insights doesn't seem to capture the newlines 🤔